pkg/util/containerd/namespaces.go

```diff
@@ -17,9 +19,33 @@ import (
 // "containerd_namespace" option has been set, it returns that namespace.
 // Otherwise, it returns all of them.
 func NamespacesToWatch(ctx context.Context, containerdClient ContainerdItf) ([]string, error) {
-	if namespace := config.Datadog.GetString("containerd_namespace"); namespace != "" {
-		return []string{namespace}, nil
+	if namespaces := config.Datadog.GetString("containerd_namespace"); namespaces != "" {
+		return strings.Split(namespaces, ","), nil
 	}
 
 	return containerdClient.Namespaces(ctx)
 }
+
+// FiltersWithNamespaces returns the given list of filters adapted to take into
+// account the namespaces that we need to watch.
+// For example, if the given filter is `topic=="/container/create"`, and the
+// namespace that we need to watch is "ns1", this function returns
+// `topic=="/container/create",namespace=="ns1"`.
+func FiltersWithNamespaces(filters []string) []string {
+	namespaces := strings.Split(config.Datadog.GetString("containerd_namespace"), ",")
```

Comment:
It would be better to give the list of namespaces as parameters

Reply:
I like the current option because this way callers don't need to know what's the config option that controls the namespaces to watch, and don't need to worry about how to parse it.